DataFormats/CTPPSReco/src/classes_def.xml

```diff
@@ -81,6 +81,16 @@
   <class name="edm::DetSetVector<CTPPSPixelCluster>"/>
   <class name="edm::Wrapper<edm::DetSetVector<CTPPSPixelCluster> >"/>
 
+  <class name="CTPPSPixelRecHit" ClassVersion="4">
+    <version ClassVersion="3" checksum="4187113353"/>
+    <version ClassVersion="4" checksum="2784452215"/>
```

Comment:
please stick to just one version. So, remove the current version 3 and then rename what is now v4 to v3

Reply:
done